examples/wflow_piave_clip/wflow_sbm.toml

```diff
@@ -117,6 +117,7 @@ length = "wflow_riverlength"
 n = "N_River"
 slope = "RiverSlope"
 width = "wflow_riverwidth"
+bankfull_elevation = "RiverZ"
```

Comment:
What about `bankfull_depth` here? 

Here `RiverZ` is now used as `bankfull_elevation`. This is probably for the 1-D local inertial approach for river flow (without the 2-D part)? Also related to the previous comment, when using the 1-D/2-D local inertial approach, is it fine to use `RiverZ` in combination with `FloodplainZ` for overland flow? Would be good to add a bit more detail to the docs.

The toml file is probably not changed when using the local inertial approach for wflow_sbm, right? I think this is fine, it is explained in the Wflow docs:
```toml
[model]
river_routing = "local-inertial"
land_routing = "local-inertial"
```

Reply:
`bankfull_depth` is currently not in the deafult toml file. Only if the `river_geom_fn` argument of the `WflowModel.setup_rivers` method is provided the `wflow_riverdepth` is calculated and the `bankfull_depth` map set in the toml file.

The  toml file is not automatically changed to use the local inertial approach. To use the local-inertial method it's best to set this option in the toml file through the `WflowModel.setup_config` method or directly in the toml file.

Reply:
See reply to earlier comments. We also set more wflow config options depending on the choice of routing.